addendum.dict

```diff
@@ -172,7 +178,7 @@ Ingolstadt	𐑦𐑙𐑜𐑩𐑤𐑖𐑑𐑭𐑑
 inspo	𐑦𐑯𐑕𐑐𐑴
 Instagram	𐑦𐑯𐑕𐑑𐑩𐑜𐑮𐑨𐑥
 interp	𐑦𐑯𐑑𐑻𐑐
-Iphone	𐑲𐑓𐑴𐑯
+iPhone	𐑲𐑓𐑴𐑯
```

Comment:
Capitalisation in this file relates to the use of the namer dot, rather than how it is conventionally spelled, so this change can be removed from the pull request.